portability-transfer/src/test/java/org/datatransferproject/transfer/JobPollingServiceTest.java

```diff
@@ -134,6 +151,7 @@ public void pollingLifeCycle() throws Exception {
             .setImportService("DummyImportService")
             .setAndValidateJobAuthorization(
                 JobAuthorization.builder()
+                    .setEncryptionScheme("cleartext")
```

Comment:
can you document somewhere that cleartext should be used for debug purposes only?

Reply:
Sure, I added it in the UI console.